sympy/solvers/ode/single.py

```diff
@@ -2207,7 +2207,7 @@ def _get_simplified_sol(self, sol, collectterms):
         for i, reroot, imroot in collectterms:
             sol = collect(sol, x**i*exp(reroot*x))
         sol = powsimp(sol)
-        return [Eq(f(x), sol)]
+        return Eq(f(x), sol)
```

Comment:
Returning anything other than a list should fail here. Is there some code somewhere else that checks for this? Why didn't this fail before?

Reply:
Currently `test_single.py` handles both type of solutions with list as well as without as some of the solvers are yet to be refactored. After refactoring I will remove the conditional part from test suite so that it should expect list of solution from every  solver.